<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TopBar</class>
 <widget class="QWidget" name="TopBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>80</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLabel" name="Logo">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../res.qrc">:/resource/image/logo.ico</pixmap>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>V1.2.0</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="pushButton_3">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../res.qrc">
          <normaloff>:/resource/image/service.png</normaloff>:/resource/image/service.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnSetting">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../res.qrc">
          <normaloff>:/resource/image/setting.png</normaloff>:/resource/image/setting.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnMinium">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../res.qrc">
          <normaloff>:/resource/image/minium.png</normaloff>:/resource/image/minium.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnClose">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../res.qrc">
          <normaloff>:/resource/image/close.png</normaloff>:/resource/image/close.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../res.qrc"/>
 </resources>
 <connections/>
</ui>
